_posts/2021-05-16-overview-of-architecture-review-processes.md

```diff
@@ -0,0 +1,253 @@
+  1. Preparation Phase: The ARB selects a review team and a review leader. Togehter with the project team, they determine the number of reviews and the initial review’s date and agenda. The review team, project team, review leader, and review angel verify that the project that should be reviewed has a clearly defined problem statement driving the architecture and an appropriate documentation as a basis for the review.
+  1. Review Meeting Phase: During the review, the members of the project present the mentioned problem statement and how the proposed architecture solves it. The reviewers ask further questions in order to figure out issues they believe could make the project an incomplete or inadequate solution to the problem. Finally, the review team meets privately and generates a report. The report is presented to the project team. Furthermore, management alerts can be raised, etc.
+  1. Follow-Up Phase: The review team delivers the report with the review’s findings to the project team within certain time frame such as 15 days after the review meetings are done. The project team (and, if a management alert has been raised, the management) must respond within two weeks.
+
+For more details, we refer to {% cite Maranzano2005 %}.
+
+### Artifacts
+
+As artifacts, Maranzano et al. mention three different types of artifacts {% cite Maranzano2005 %}:
+  1. There are checklists for architects to prepare for the review and checklists for the reviewers to do the review and figure out more of the architecture. The checklists serve as a collection of organizational knowledge and should be maintained.
+  1. There is input to the review. This can be documentation of the system requirements, functional requirements, or informal documentation. Maranzano et al. mention specifically that the architecture should be designed based on a clear problem statement tackling the functional and qualitative requirements, the costs, and the timeline. A document describing these points may be a specific design document or something like that.
+  1. There is output from the review such as the review report, an optional management alert letter, an optional set of issues, etc.
+
+### Conclusion
+
+The "Classical" Architecture Review process is a well-established and formalized process.
+It is (still) widely used.
+Especially, bigger companies or, at least, companies with dedicated architects often make use of a process explained by Maranzano et al. in some variation.
+
+On the one hand, the formalized process as well as the clear roles within the process help to achieve the goals of architecture reviews.
+Also, there are different options to vary the process in order to, for example, have less parties involved or reduce the overhead.
+In some companies, there is, for instance, no ARB or review team but an architecture guild which is performing architecture reviews instead.
```

Comment:
Maybe linking the word "guild" to an explanation (e.g. community of practice). Why? Not everyone might know what this is.